test/integration/api-schema/download-json-schemas.sh

```diff
@@ -1,21 +1,24 @@
-
 #!/usr/bin/env bash
 
 set -ex
 if ! [ -n "$1" ]; then
     echo "USAGE: ./download-json-schemas.sh /path/to/folder"
     echo ""
     echo "Downloads the APM Server Schema files to the specified folder"
+    echo ""
+    echo "Sources from: https://codeload.github.com/elastic/apm-server/"
     exit 1
 fi
 
+# download_schema [/path/to/download/to] [branch]
```

Comment:
nit: This function is written to *require* both those args, right. Could we write that as:

```suggestion
# download_schema </path/to/download/to> <branch>
```

It may be I'm showing a bias to use `[...]` syntax for optional args.